beaglebone/test/project.ini

```diff
@@ -0,0 +1,4 @@
+[build]
```

Comment:
If we can, we should delete these test files (beaglebone/test and beaglebone/test1).